### ISSUE TITLE
Shortcut to Jump to Next/Previous Rehearsal Mark

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#28357]
### Your idea

Customisable Shortcuts to navigate to the next and previous rehearsal marks. 

### Problem to be solved

Currently it is really hard for blind users to find ways to get around a score and find specific points.  One of the few ways other than memorising bar numbers is to use Rehearsal marks.  This works fine if you know what letter you want to jump to and if they are sequential.  But if trying to use these more extensively to navigate for example a book of theory exercises then these might be question numbers or something other than letters and so you have to know what you are looking for before you can get there.  The time line is also not accessible currently.  But the easiest way to move through a score would be to have a shortcut to move to the next/previous rehearsal mark.  I can then for example place rehearsal marks at each point a student needs to correct something and quickly get back to them.

I think many sighted users would find this a really quick way to navigate also as I'm told the timeline is quite bulky on screen and often not used much.

### Prior art

e

### Checklist

- [x] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [x] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by jeetee** [comment]
I note that a "go to next/prev frame" action already exists as well and it got me wondering whether a more generic "go to next/prev element of same type" could be an approach to satisfy both behaviors in a common flow?
(Though you'll still need/want to navigate to the first element of that type somehow..)

**Comment by jmwmusic** [comment]
Yes a find next/previous element has already been requested #20055 and suggestions that #23890 could be used for this.  However, that might be a bigger project so in the short term the two most requested are to jump to next/previous section #23012 which was apparently present in v.3 (especially if there are multiple songs in a book) and rehearsal marks for large scores and exercise books.
